aten/src/ATen/Dispatch.h

```diff
@@ -44,5 +44,22 @@ R dispatch_floating_types(const Type& the_type, const char *name, Args&&... args
   }
 }
 
+#define DISPATCH_ALL_FLOATING_TYPES(TYPE, NAME, F...)                             \
```

Comment:
Windows CI wasn't working properly earlier, but it seems that MSVC complains about `error C2010: '.': unexpected in macro formal parameter list`. Any suggestions on how to fix it?

Reply:
I think we can replace `F...` with `...` and `F()` with `__VA_ARGS__()`